x/batch/msg.go

```diff
@@ -23,7 +23,7 @@ func Validate(msg Msg) error {
 	}
 
 	if len(l) > MaxBatchMessages {
-		return errors.ErrTooLarge()
+		return errors.ErrInvalidInput.New("transaction is too large")
```

Comment:
Can you add the number of message and the maximum limit here? This makes it easier to correct client side